index.html

```diff
@@ -2,95 +2,186 @@
+                <p class="section-pay__head--p button-font-color-blue">Designed For Engineers</p>
+                <h2 class="section-pay__head--title section-API__head--title">
+                    The world’s best and most intuitive APIs
+                </h2>
+                <p class="paragraph-white">
+                    We abstract the hard stuff away so your teams don't
+                    can focus on building good technology, instead of
+                    spending time and money reinventing the wheel.
+                </p>
+                <div class="main-txt-img__buttons">
+                    <button class="main-txt-img__buttons--black section-API__button-blue">Start now
+                        <img src="assets/arrow-right-light.svg" alt="arrow-right-light" />
+                    </button>
                 </div>
-                <div class="section-pay__body">
-                    <img class="card" src="assets/credit-card.png" alt="credit-card-img">
-                    <img class="slack" src="assets/slack-invoice.png" alt="slack-img">
-                    <img class="phone" src="assets/phone.png" alt="phone-img">
-                    <img class="balance" src="assets/dash.png" alt="balances-img">
+                <div class="section-API__RIGHT--2columns">
+                    <div class="section-API__RIGHT__column-L">
+                        <img class="icons-img" src="assets/tools-icon.png" alt="tools-icon img">
+                        <h3>Tools for all stacks</h3>
+                        <p>We offer front end and back end libraries in some of the most widely used technologies, old and new.</p>
+                        <button class="main-txt-img__buttons--hiden button-font-color-blue">
+                            See libraries
+                            <img src="assets/arrow-right-blue.svg" alt="arrow-right-blue" />
+                        </button>
+                        
+                    </div>
+                    <div class="section-API__RIGHT__column-R">
+                        <img class="icons-img" src="assets/cube-icon.png" alt="cube-icon img">
+                        <h3>Custom Integrations</h3>
+                        <p>Use integrations for systems like Shopify, WooCommerce, NetSuite, and more.</p>
+                        <button class="main-txt-img__buttons--hiden button-font-color-blue">
+                            Explore partners
+                            <img src="assets/arrow-right-blue.svg" alt="arrow-right-blue" />
+                        </button>
+                    </div>
                 </div>
-            </section>
+
+            </div>
+            <div class="section-API__LEFT">
+                <img
+                    class="code-img"
+                    src="assets/api-code.png"
+                    alt="api-code-img"
+                />
+                <img
+                    class="terminal-img"
+                    src="/assets/terminal-code.png"
```

Comment:
por este slash es que no me sale la imagen, podrías por favor removerlo?
```suggestion
                    src="assets/terminal-code.png"
```

Reply:
Mira cómo se está generando una barra blanca en el header:
![image](https://github.com/JJBerdugo/swipe/assets/100431019/67cde34f-3088-43ca-ba18-f73bdca29bff)

Y la está causando el desborde que está haciendo el fondo púrpura en la sección 3:

![image](https://github.com/JJBerdugo/swipe/assets/100431019/9a5d31f1-6535-4043-9dce-435de0f1948f)

Revisa si es posible hacer los cambios y envías a revisión nuevamente por favor!

Reply:
si me he dado cuenta de eso y de otro errores por ahi y estoy tratando de corregirlos para avanzar... si puedo contar con tu ayuda seria genial!... nuevamente gracias!